addons/google_recaptcha/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import requests
+
+from odoo import _, api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    recaptcha_public_key = fields.Char("Public Key", config_parameter='recaptcha_public_key', groups='base.group_system')
```

Comment:
rename label to: 'site key' and 'secret key' like in google
![image](https://user-images.githubusercontent.com/4848797/83128232-6ae69600-a0db-11ea-8a01-055f9009c139.png)